File: typescript/iniciante/1018.ts
import { bold, magenta } from "https://deno.land/std@0.224.0/fmt/colors.ts";

const N = Number(prompt("Digite um valor inteiro:"));
let restante = N;

const notas = [100, 50, 20, 10, 5, 2, 1];

console.log(magenta(bold(String(N))));

for (const nota of notas) {
  const qtd = Math.floor(restante / nota);
  restante = restante % nota;
  console.log(magenta(`${qtd} nota(s) de R$ ${bold(String(nota))},00`));
}

File: swift/iniciante/1018.swift
import Foundation

if let input = readLine(), let N = Int(input) {
    var restante = N

    let notas = [100, 50, 20, 10, 5, 2, 1]

    print(N)

    for nota in notas {
        let qtd = restante / nota
        restante = restante % nota
        print("\(qtd) nota(s) de R$ \(nota),00")
    }
}
